ngraph/frontend/onnx_import/src/op/resize.cpp

```diff
@@ -178,10 +279,17 @@ namespace ngraph
                     size_t axes_size = scales_shape.is_static() ? scales_shape[0].get_length()
                                                                 : data_shape.rank().get_length();
 
-                    const auto output_shape = calculate_output_shape_based_on_scales(data, scales);
+                    std::vector<int64_t> axes;
+                    for (int ax = 0; ax < axes_size; ++ax)
+                    {
+                        axes.push_back(ax);
+                    }
 
-                    return build_resize(
-                        node, output_shape, AxisSet(common::get_monotonic_range(axes_size)));
+                    const auto axes_const = default_opset::Constant::create(
+                        ngraph::element::i64, Shape({axes.size()}), axes);
+                    const auto output_shape = calculate_output_shape_based_on_scales(data, scales);
+                    return {std::make_shared<ngraph::op::v4::Interpolate>(
+                        data, output_shape, scales, axes_const, attrs)};
```

Comment:
As we discuss on [Interpolate-4 spec review](https://github.com/openvinotoolkit/openvino/pull/1035#issuecomment-656683783) to make `axes` input optional,
can we use in onnx importer `ngraph::op::v4::Interpolate` constructor without the `axes` arg and remove from onnx import the code generating axes input? (same comment for Upsample)

```
op::v4::Interpolate::Interpolate(const Output<Node>& image,
                                 const Output<Node>& output_shape,
                                 const Output<Node>& scales,
                                 const op::v4::Interpolate::InterpolateAttrs& attrs)
```

Reply:
Done.